dev-lua/lua-argparse/lua-argparse-0.6.0.ebuild

```diff
@@ -0,0 +1,46 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit toolchain-funcs
+
+DESCRIPTION="Command line argument parser for the Lua Programming Language"
+HOMEPAGE="https://github.com/mpeterv/argparse"
+SRC_URI="https://github.com/mpeterv/argparse/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="doc luajit"
+
+RDEPEND="
+	>=dev-lang/lua-5.1:=
+	luajit? ( dev-lang/luajit:2 )
+"
+DEPEND="${RDEPEND}"
+
+S="${WORKDIR}/${PN//lua-/}-${PV}"
+
+DOCS=( README.md CHANGELOG.md )
+
+src_compile() {
+	if use doc; then
+		sphinx-build docsrc html || die
```

Comment:
you run sphinx but do not depend on it in any way.
you have to add doc useflag gated sphinx dependency.

Reply:
fixed